lib/Reader/reader.cpp

```diff
@@ -8015,6 +8027,8 @@ void ReaderBase::msilToIR(void) {
     ++it;
   }
 
+  popDebugScope();
```

Comment:
It seems odd to me that `popDebugScope()` is here in the base class but `push` is only in the derived class. I would have expected them both to be at the same level...